src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> node = new Node<>(value);
+
+        if (first == null) {
+            first = node;
+            size++;
+
+            return;
+        }
+
+        if (last == null) {
+            last = node;
+            last.prev = first;
+            first.next = last;
+            size++;
+
+            return;
+        }
+
+        last.next = node;
+        node.prev = last;
+        last = node;
+        size++;
```

Comment:
Please, remove empty lines on this method

Reply:
You can use `if/else` structure instead of separate `if`, and remove `return` from `if`